docs/en/Configuring-Alluxio-with-secure-HDFS.md

```diff
@@ -29,7 +29,8 @@ of Alluxio source code.
 
 * Modify the `hadoop.version` tag defined in `${ALLUXIO_HOME}/pom.xml`. E.g., to work with Hadoop
 `2.6.0`, modify this pom file to set "`<hadoop.version>2.6.0</hadoop.version>`" instead of
-"`<hadoop.version>2.2.0</hadoop.version>`". Then recompile the source using maven.
+"`<hadoop.version>2.2.0</hadoop.version>`". Then recompile the source using maven. 
+To make compiling faster, you can add `-DskipTests` option to skip unit tests.
```

Comment:
previous comment from @calvinjia : 
Q: Let's add the skip find bugs / checkstyle as well?
A: Hmm, test should be the bottleneck to build Alluxio. I think find bugs/ checkstyle should be pretty fast.
In other docs, only -DskipTests is applied. Do we want keep the sample command elegant, with as few options as possible?